test/number-is-prime/index.js

```diff
@@ -0,0 +1,16 @@
+var test = require('../util/test')(__filename);
+var isPrime = require('../../packages/number-is-prime');
+
+test('number is prime', function(t) {
```

Comment:
also test for `0` and one large number, non integer numbers, other types, null/undefined